app/services/proofing/resolution/result_adjudicator.rb

```diff
@@ -2,20 +2,24 @@ module Proofing
   module Resolution
     class ResultAdjudicator
       attr_reader :resolution_result, :state_id_result, :device_profiling_result,
-                  :double_address_verification
+                  :double_address_verification, :residential_resolution_result, :same_address_as_id
 
       def initialize(
-        resolution_result:,
-        state_id_result:,
+        resolution_result:, # IV
```

Comment:
is iv identity verification?

Reply:
IV stands for InstantVerify.  I can change that.  I can see how the acronym IV is unclear.